charts/opensearch/ci/ci-ingress-class-name-values.yaml

```diff
@@ -318,12 +320,21 @@ terminationGracePeriod: 120
 
 sysctlVmMaxMapCount: 262144
 
+startupProbe:
+  enabled: false
```

Comment:
Hey @TheAlgo, thanks for the fix, do we still need this `enabled` flag ?

Reply:
Not required, I missed this CI value , removing it